### PR TITLE
Update docs about how SRE access OSD clusters

### PR DESCRIPTION
**Pull request by wanghaoran1988**

### CODE REVIEW

File: modules/policy-identity-access-management.adoc [hunk 1]
```diff
@@ -13,9 +13,7 @@ For a list of the available subprocessors, see the link:https://access.redhat.co
...
-
-SREs access private clusters using an encrypted tunnel through a hardened SRE Support Pod running in the cluster. Connections to the SRE Support Pod are permitted only from a secured Red Hat network using an IP allow-list. In addition to the cluster authentication controls described above, authentication to the SRE Support Pod is controlled using SSH keys. SSH key authorization is limited to SRE staff and automatically synchronized with Red Hat corporate directory data. Corporate directory data is secured and controlled by HR systems, including management review, approval, and audits.
+SREs access {product-title} clusters through a proxy running in front of OSD clusters, The proxy will mint a Service Account in OSD cluster for SREs when they try to login to the cluster. SREs must authenticate as individuals to ensure auditability. All authentication attempts are logged to a Security Information and Event Management (SIEM) system.
```
Comment (chopraapooja) [inline modules/policy-identity-access-management.adoc:16]: @wanghaoran1988 Is it worth clarifying to the customer that we don't use the web console or usual command-line login? I have seen some queries around this in the past from customers as secondary

Comment (wanghaoran1988) [reply modules/policy-identity-access-management.adoc]: @chopraapooja Updated

**Comment by vikram-redhat** [comment]
@pneedle-rh can you take a look when you are in next, and merge if ok?

File: modules/policy-identity-access-management.adoc [hunk 2]
```diff
@@ -13,9 +13,7 @@ For a list of the available subprocessors, see the link:https://access.redhat.co
...
-
-SREs access private clusters using an encrypted tunnel through a hardened SRE Support Pod running in the cluster. Connections to the SRE Support Pod are permitted only from a secured Red Hat network using an IP allow-list. In addition to the cluster authentication controls described above, authentication to the SRE Support Pod is controlled using SSH keys. SSH key authorization is limited to SRE staff and automatically synchronized with Red Hat corporate directory data. Corporate directory data is secured and controlled by HR systems, including management review, approval, and audits.
+SREs access {product-title} clusters through a proxy running in front of OSD clusters, The proxy will mint a Service Account in OSD cluster for SREs when they try to login to the cluster. As there is no IdentityProvider setup on OSD clusters, we are not access the cluster console dirrectly, instead we are running a local console container to access the proxy when needed. SREs must authenticate as individuals to ensure auditability. All authentication attempts are logged to a Security Information and Event Management (SIEM) system.
```
Comment (pneedle-rh) [inline modules/policy-identity-access-management.adoc:16]: @wanghaoran1988 thank you for this docs contribution. You could update this line to something like the following, to introduce minimalism and some additional consistency with the docs guidelines:

"SREs access {product-title} clusters through a proxy. The proxy mints a service account in an OSD cluster for the SREs when they login. As no identity provider is configured for OSD clusters, SREs access the proxy by running a local console container. SREs do not access the cluster console directly. SREs must authenticate as individual users to ensure auditability. All authentication attempts are logged to a Security Information and Event Management (SIEM) system."

Comment (pneedle-rh) [reply modules/policy-identity-access-management.adoc:16]: @wanghaoran1988 you could also update the reference to "cluster console" to be explicit about the type of console that is being referred to. For example, if it is the web console that is being used, you could state "cluster web console" or something similar.

File: modules/policy-identity-access-management.adoc [hunk 3]
```diff
@@ -25,7 +23,7 @@ Red Hat SRE adheres to the principle of least privilege when accessing {product-
 
 * SRE admin access through the Red Hat corporate SSO with normal two-factor authentication and no privileged elevation.
 
-* OpenShift elevation, which is a manual elevation using Red Hat SSO. It is limited to 2 hours, is fully audited, and requires management approval.
+* OpenShift elevation, which is a manual elevation using Red Hat SSO. It is fully audited, and requires management approval for every operations SRE made.
```
Comment (pneedle-rh) [inline modules/policy-identity-access-management.adoc:26]: You could remove the comma after "audited" and make "operations" singular. For example:

"It is fully audited and management approval is required for every operation SREs make."

**Review by pneedle-rh — COMMENTED** [review]
@wanghaoran1988 thank you for submitting this PR. I have completed my review. Please let me know if you have any questions about my suggestions. Thanks!

**Comment by wanghaoran1988** [comment]
@pneedle-rh Comments addressed, could you take a look again?

**Comment by pneedle-rh** [comment]
@wanghaoran1988 thank you for addressing the suggestions. Your updates look good to me.

Can you please squash your commits into one? Thanks!

**Comment by wanghaoran1988** [comment]
@pneedle-rh squashed now.

File: modules/policy-identity-access-management.adoc [hunk 4]
```diff
@@ -13,9 +13,7 @@ For a list of the available subprocessors, see the link:https://access.redhat.co
...
-
-SREs access private clusters using an encrypted tunnel through a hardened SRE Support Pod running in the cluster. Connections to the SRE Support Pod are permitted only from a secured Red Hat network using an IP allow-list. In addition to the cluster authentication controls described above, authentication to the SRE Support Pod is controlled using SSH keys. SSH key authorization is limited to SRE staff and automatically synchronized with Red Hat corporate directory data. Corporate directory data is secured and controlled by HR systems, including management review, approval, and audits.
+SREs access {product-title} clusters through a proxy. The proxy mints a service account in an OSD cluster for the SREs when they login. As no identity provider is configured for OSD clusters, SREs access the proxy by running a local web console container. SREs do not access the cluster web console directly. SREs must authenticate as individual users to ensure auditability. All authentication attempts are logged to a Security Information and Event Management (SIEM) system.
```
Comment (pneedle-rh) [inline modules/policy-identity-access-management.adoc:16]: @wanghaoran1988 thank you for squashing the commits.

My apologies, I used "OSD" in my suggestions the other day, where I should have recommended "{product-title}". Can you please update each instance of "OSD" in this line to "{product-title}"? After that, this PR I think is ready to merge.

Please note that you will need to squash the commits again if you update by using a new commit.

Thanks!

**Comment by pneedle-rh** [comment]
I just created  https://github.com/openshift/openshift-docs/pull/38487. This is a very small fix to update one instance of "login" to the verb form in one of the lines updated in this PR.

**Pull request merged**